src/main/java/core/basesyntax/service/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dto.TransferAction;
+
+public interface OperationStrategy {
+    void process(FruitDao fruitDao, TransferAction transferAction);
```

Comment:
let's better not pass dao as a parameter, pass it to the constructor where you create handlers
```suggestion
    void process(Transaction transaction);
```